runtime/v2/builder.go

```diff
@@ -119,6 +124,30 @@ func (a *AppBuilder[T]) Build(opts ...AppBuilderOption[T]) (*App[T], error) {
 	}
 	a.app.stf = stf
 
+	v := a.viper
+	home := v.GetString(FlagHome)
+
+	storeOpts := rootstore.Options{}
+	if v != nil {
```

Comment:
nit: the nil check should be earlier or removed. We would already have a nil pointer exception at L128 (`v.GetString`)